### PR TITLE
Add dummy app. Tests

### PR DESCRIPTION
**Pull request by ksin**
Additions:
- dummy app
- basic integration test
- unit tests for services and initializers

TODO:
- more integration tests and tests on component (ie: click calls featherEditor.launch() )

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return {};
+  var ENV = {
+    contentSecurityPolicy: {
+      'default-src': "'none'",
+      'style-src': "'self' 'unsafe-inline' 'unsafe-eval' http://feather.aviary.com/3.1.0.284/css/feather_theme_aviary_dark.css http://feather.aviary.com/3.1.0.284/css/feather_theme_aviary_light.css",
```
Comment (mfamilia) [inline config/environment.js:7]: Why are these needed?

Comment (ksin) [inline config/environment.js:7]: Console vomits at un-whitelisted content because of the CSP addon

Comment (ksin) [inline config/environment.js:7]: This doesn't completely solve the problem, but makes it a little cleaner.

Comment (mfamilia) [inline config/environment.js:7]: So this is due to what is loaded by the bower installed aviary-editor lib?

Comment (mfamilia) [inline config/environment.js:7]: just wondering if this is still needed since the aviary-editor lib is baked into the addon

Comment (ksin) [inline config/environment.js:7]: this is unrelated to bower.
